data/RTTR/CAMPAIGNS/roman/MISS200.LUA

```diff
@@ -0,0 +1,517 @@
+            [GD_TONGS     ] = 0,
+            [GD_HAMMER    ] = 0,
+            [GD_AXE       ] = 0,
+            [GD_SAW       ] = 0,
+            [GD_PICKAXE   ] = 0,
+            [GD_SHOVEL    ] = 0,
+            [GD_CRUCIBLE  ] = 0,
+            [GD_RODANDLINE] = 0,
+            [GD_SCYTHE    ] = 0,
+            [GD_WATER     ] = 12,
+            [GD_CLEAVER   ] = 0,
+            [GD_ROLLINGPIN] = 0,
+            [GD_BOW       ] = 0,
+            [GD_BOAT      ] = 0,
+            [GD_SWORD     ] = 0,
+            [GD_IRON      ] = 0,
+            [GD_FLOUR     ] = 0,
+            [GD_FISH      ] = 2,
+            [GD_BREAD     ] = 4,
+            [GD_SHIELD    ] = 0,
+            [GD_WOOD      ] = 0,
+            [GD_BOARDS    ] = 15,
+            [GD_STONES    ] = 12,
+            [GD_GRAIN     ] = 0,
+            [GD_COINS     ] = 0,
+            [GD_GOLD      ] = 0,
+            [GD_IRONORE   ] = 0,
+            [GD_COAL      ] = 20,
+            [GD_MEAT      ] = 2,
+            [GD_HAM       ] = 0
+        })
+
+        -- people
+        rttr:GetPlayer(p):AddPeople({
+        [JOB_HELPER             ] = 40,
+        [JOB_WOODCUTTER         ] = 8,
+        [JOB_FISHER             ] = 4,
+        [JOB_FORESTER           ] = 4,
+        [JOB_CARPENTER          ] = 3,
+        [JOB_STONEMASON         ] = 4,
+        [JOB_HUNTER             ] = 4,
+        [JOB_FARMER             ] = 6,
+        [JOB_MILLER             ] = 3,
+        [JOB_BAKER              ] = 3,
+        [JOB_BUTCHER            ] = 3,
+        [JOB_MINER              ] = 8,
+        [JOB_BREWER             ] = 0,
+        [JOB_PIGBREEDER         ] = 4,
+        [JOB_DONKEYBREEDER      ] = 0,
+        [JOB_IRONFOUNDER        ] = 4,
+        [JOB_MINTER             ] = 4,
+        [JOB_METALWORKER        ] = 4,
+        [JOB_ARMORER            ] = 4,
+        [JOB_BUILDER            ] = 4,
+        [JOB_PLANER             ] = 4,
+        [JOB_GEOLOGIST          ] = 0,
+        [JOB_PRIVATE            ] = 8,
+        [JOB_PRIVATEFIRSTCLASS  ] = 0,
+        [JOB_SERGEANT           ] = 0,
+        [JOB_OFFICER            ] = 0,
+        [JOB_GENERAL            ] = 0,
+        [JOB_SCOUT              ] = 8,
+        [JOB_SHIPWRIGHT         ] = 0,
+        [JOB_PACKDONKEY         ] = 15,
+        [JOB_CHARBURNER         ] = 0
+        })
+    end
+end
+
+
+-------------------------------- mission events -------------------------------
+function onGameFrame(gf)
+    -- events called by GetBuildingCount(), GetWareCount() or GetPeopleCount()
+    MissionEvent(2)
+    MissionEvent(3)
+    MissionEvent(6)
+    MissionEvent(12)
+    MissionEvent(13)
+    MissionEvent(14)
+    MissionEvent(15)
+end
+
+function onOccupied(p, x, y)
+    -- only check human player
+    if(p ~= 0) then
+        return
+    end
+
+    if(     (x == 34) and (y == 28) ) then MissionEvent(4) -- RttR calls onOccupied once a pole is on that spot, SettlersII called the function once the spot is really owned
```

Comment:
> RttR calls onOccupied once a pole is on that spot, SettlersII called the function once the spot is really owned

what do you mean by that? whats exactly the difference?

Reply:
I copied the lua files from @Spikeone repositiory. The difference is here when the border is on the position the mission is won and in the original settler the mission is won when the gate in witgin the teritory. I don't think this is a problem for now.